rekognition_tutorial/short_sa_rekognition_tutorial.ipynb

```
@@ -0,0 +1,1222 @@
+    "    :param rek_client: The Amazon Rekognition Custom Labels Boto3 client.\n",
+    "    :param project_arn: The ARN of the project in which you want to create a dataset.\n",
+    "    :param dataset_type: The type of the dataset that you want to create (train or test).\n",
+    "    \"\"\"\n",
+    "    try:\n",
+    "        #Create the dataset\n",
+    "        print(f\"Creating empty {dataset_type} dataset for project {project_arn}\")\n",
+    "\n",
+    "        dataset_type = dataset_type.upper()\n",
+    "\n",
+    "        response = rek_client.create_dataset(ProjectArn = project_arn,\n",
+    "                                             DatasetType = dataset_type)\n",
+    "\n",
+    "        dataset_arn = response['DatasetArn']\n",
+    "\n",
+    "        print(f\"dataset ARN: {dataset_arn}\")\n",
+    "\n",
+    "        finished = False\n",
+    "        while finished == False:\n",
+    "\n",
+    "            dataset = rek_client.describe_dataset(DatasetArn=dataset_arn)\n",
+    "\n",
+    "            status = dataset['DatasetDescription']['Status']\n",
+    "            \n",
+    "            if status == \"CREATE_IN_PROGRESS\":\n",
+    "                \n",
+    "                print((f\"Creating dataset: {dataset_arn} \"))\n",
+    "                time.sleep(5)\n",
+    "                continue\n",
+    "\n",
+    "            if status == \"CREATE_COMPLETE\":\n",
+    "                print(f\"Dataset created: {dataset_arn}\")\n",
+    "                finished = True\n",
+    "                continue\n",
+    "\n",
+    "            if status == \"CREATE_FAILED\":\n",
+    "                raise Exception (f\"Dataset creation failed: {status} : {dataset_arn}\")\n",
+    "                \n",
+    "            \n",
+    "        return dataset_arn\n",
+    "       \n",
+    "    except ClientError as err:  \n",
+    "        print(f\"Could not create dataset: {err.response['Error']['Message']}\")\n",
+    "        raise"
```

maybe raise an exception too?